ament_index_cpp/src/get_resources.cpp

```diff
@@ -46,10 +46,18 @@ get_resources(const std::string & resource_type)
     }
     dirent * entry;
     while ((entry = readdir(dir)) != NULL) {
-      // ignore folders starting with a dot
+      // ignore directories/files starting with a dot
       if (entry->d_name[0] == '.') {
         continue;
       }
+
+      // ignore directories
+      auto subdir = opendir((path + "/" + entry->d_name).c_str());
```

Comment:
I was just copying the behaviour [here](https://github.com/dhood/ament_index/blob/master/ament_index_cpp/src/get_resources.cpp#L46) but if you want I can add warnings in both places?